src/fontra/views/editor/editor.js

```diff
@@ -452,6 +454,47 @@ export class EditorController {
     this.initSidebarTextEntry();
   }
 
+  initSidebarGutters() {
+    let initialWidth;
+    let initialPointerCoordinateX;
+    let sidebarResizing;
+    let growDirection;
+    document.body.addEventListener("pointerdown", (event) => {
```

Comment:
I know there are two gutter elements, but I wonder: wouldn't it be better to add the event listeners to the gutter elements instead of the body? Then at least some of the event dispatching is done by the browser, and not by our own JS code.

Reply:
Hmm maybe I could register the event when the pointer is down, and unregister when the pointer is up. The events will be added only when a resize is initiated.

Reply:
I will submit a patch.

Reply:
I did.